### ISSUE TITLE
Add the instruction of the improvement...

### ISSUE THREAD

**Issue by tjx666** [dunstontc/dark-plus-syntax#6]
I think many people would like to know where this theme improved as me.

**Comment by dunstontc** [comment]
@tjx666 You're asking for a list of differences between the standard, built-in `Dark+ (default dark)` and this theme `dark-plus-syntax`, correct? Just want to make sure I understood you correctly.

**Comment by tjx666** [comment]
yes

**Comment by tjx666** [comment]
Yes @dunstontc Maybe is also called features.

**Comment by tcd** [comment]
@tjx666 I've added a [Contributing](https://github.com/dunstontc/dark-plus-syntax#contributing) section in the readme with an explanation of how the project is structured and how to make changes to it. Take a look and let me know if that's sufficient.

**Comment by lowercase00** [comment]
@tcd @dunstontc  hey mate, I followed the instructions, but after running the build command and the debugger I could see any difference. Any other specific step to make it available on the color picker? Or it is the modified (new) version the same theme on the dropdown? I'm trying to change Python type hint for custom types, I think I found the tokens, but couldn't find a way to test if it worked. Thanks!
